force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +99,20 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
         // Create a set to store the unique words
+        Set<String> stringSet = new Set<String>();
+        List<String> words = sentence.split(' ');
+        String results = ' ';
 
         // Add each word to the set
+        for(String word : words){
+            stringSet.add(word);
+        }
 
-        // Join the unique words back into a sentence
-
-        // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+       // Join the unique words back into a sentence
+       // Return the sentence with unique words
+      
+        return String.join(new List<String>(stringSet), ' '); 
```

Comment:
This works. But there is no need for iteration and it looks like that variable result is not used.  You can pass a list directly into a Set constructor. 

```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```